core-feature-pack/common/pom.xml

```diff
@@ -212,6 +212,21 @@
             <artifactId>slf4j-api</artifactId>
         </dependency>
 
+        <dependency>
```

Comment:
Should these new dependencies be listed in `core-feature-pack/common/src/license/core-feature-pack-common-licenses.xml`?